packages/components/src/components/CoinLogo/index.native.tsx

```diff
@@ -31,12 +31,20 @@ const Logo = styled.Image``;
 
 interface Props {
     network: string;
+    height: number | string;
 }
 
-const CoinLogo = ({ network, ...rest }: Props) => {
+const CoinLogo = ({ network, height = 50, ...rest }: Props) => {
     return (
-        // eslint-disable-next-line import/no-dynamic-require(
-        <Logo source={LOGOS.xtz} {...rest} />
+        <Logo
+            source={LOGOS[network]}
+            style={{
+                width: height,
```

Comment:
this assumes all coin logos are rectangular which is not true.
Couldn't it work just like html img tag where you can specify width/height and second dimension (height/width) is automatically calculated

Reply:
This is the only way i could resolve these logos in a way that you can specify the size and it's fit into that size. 
It's because in react native there is nothing like `height: auto` or `width: auto` for images so you need to use imageResizeMode which works something like positioning an image as background inside div in web.
So it's basically a square view with image with resizeMode: contain

Reply:
yeah, my bad. I meant square not rectangular. that is fine. my point is that you can probably omit `width: height` and it will work the same.

Reply:
I need to set booth width and height so the image is not cropped because of the resizeMode: contain;
So it won't work the same without `width: height`;
Maybe the prop should be called `size` but then we need to add it to the non-native component.